lasagne/utils.py

```diff
@@ -205,3 +205,82 @@ def compute_norms(array, norm_axes=None):
     norms = np.sqrt(np.sum(array**2, axis=sum_over))
 
     return norms
+
+
+def create_param(spec, shape, name=None):
+    """
+    Helper method to create Theano shared variables for layer parameters
+    and to initialize them.
+
+    Parameters
+    ----------
+
+    spec : numpy array, Theano shared variable, or callable
+        One of three things:
+            * a numpy array with the initial parameter values
+            * a Theano shared variable representing the parameters
```

Comment:
I think for the markup to work there must be an new line before the list, and the list should not be intended (i.e., it should start on the same level as "One of three things").